src/components/ToggleButton/ToggleButton.js

```diff
@@ -40,6 +44,7 @@ function IconOnly({ icon, iconProps, title, onClick, className }) {
       title={title}
       onClick={onClick}
       className={classNames(className, 'icon-toggle')}
+      data-testid={props['data-testid']}
```

Comment:
Maybe `testId` instead?

Reply:
I kind of like having the consistent `data-testid`. To answer your question, it needs to be passed in because I'm referencing it in unit tests

Reply:
It's a React component prop name, it should not have a dash in it.

Reply:
will fix